tls/s2n_server_cert_request.c

```diff
@@ -97,10 +98,32 @@ static int s2n_set_cert_chain_as_client(struct s2n_connection *conn)
     return 0;
 }
 
-int s2n_client_cert_req_recv(struct s2n_connection *conn)
+int s2n_server_cert_req_recv(struct s2n_connection *conn)
 {
     struct s2n_stuffer *in = &conn->handshake.io;
 
+    if (conn->actual_protocol_version >= S2N_TLS13) {
```

Comment:
This shares no code in common with the tls1.2 version. It should probably be a separate handler.